commit-to-output.js

```diff
@@ -92,7 +98,11 @@ function toStringMarkdown (data) {
 }
 
 function toStringMessageOnly (data) {
-  return `  * ${data.summary.trim()}`
+  let cveData = ''
+  if (data.cveId) {
+    cveData = `${data.cveId} - `
+  }
+  return `  * ${cveData}${data.summary.trim()}`
```

Comment:
how about simplifying this, also the parens to match the full markdown version:

```suggestion
  return `  * ${data.cveId ? '(' + data.cveId + ') ' : ''}${data.summary.trim()}`
```